backend/resources/contact.py

```diff
@@ -23,9 +23,8 @@ def post(self):
             reply_to=new_msg["email"],
             recipients=["cfp@codeforpoznan.test"],
         )
-        msg.body = "Nowa wiadomość od {}, nr tel: {} \nTreść:\n {}".format(
-            new_msg["name"], new_msg["phone"], new_msg["content"]
-        )
+        msg.body = f"Nowa wiadomość od {new_msg['name']}, \
```

Comment:
I don't want to rule the formatting as it's very opinionated but I'd pull these values out to simpler variables, like so:
```
name, phone, content = new_msg['name'], new_msg['phone'], new_msg['content']
msg.body = f"Nowa wiadomość od {name}, nr tel: {phone} \nTreść:\n {content}"
```
Which seems bit cleaner to me. Maybe the vars shouldn't be stacked in one line too, but that's okayish imo.
Anyway, I'd like to hear the opinion of rest of the team on that.